src/hooks/useLocaleFormatter/useLocaleFormatter.ts

```diff
@@ -112,6 +112,34 @@ type useLocaleFormatterArgs = {
   fiatType?: string
 }
 
+/** Format a number as a crypto display value */
+export const numberToCrypto = (
+  num: BigNumber.Value,
+  symbol = 'BTC',
+  localeParts: LocaleParts,
+  options?: NumberFormatOptions,
+): string => {
+  if (
+    options?.truncateLargeNumbers &&
+    bnOrZero(num).toFixed(0).length > MAXIMUM_LARGE_NUMBER_CHARS
+  ) {
+    return `${truncate(bnOrZero(num).toFixed(0), { length: 16, omission: '...' })} ${symbol}`
+  }
+
+  const maximumFractionDigits =
+    options?.maximumFractionDigits !== undefined ? options.maximumFractionDigits : CRYPTO_PRECISION
+
+  const formatOptions = {
+    decimalSeparator: localeParts.decimal,
+    groupSeparator: localeParts.group,
+    groupSize: localeParts.groupSize,
+    secondaryGroupSize: localeParts.secondaryGroupSize,
+    suffix: ` ${symbol}`,
+  }
+
+  return bnOrZero(num).decimalPlaces(maximumFractionDigits).toFormat(formatOptions)
+}
+
```

Comment:
q: why do we need this?

Reply:
I need to revert this now as the handler is directly in the hook it's not needed anymore